contents/blog/north-star-metrics.md

```diff
@@ -0,0 +1,110 @@
+
+That's why we've chosen a user experience North Star metric: Discoveries.
+
+We define Discoveries as:
+
+ 1. **Analyzing any insight** (e.g. [trends](https://posthog.com/docs/user-guides/trends), [funnels](https://posthog.com/docs/user-guides/funnels), [paths](https://posthog.com/docs/user-guides/paths), and [lifecycle](https://posthog.com/docs/user-guides/lifecycle) for 10 seconds or more
+ 2. **Analyzing a [session recording](https://posthog.com/docs/user-guides/recordings)** for 10 seconds or more.
+
+### Why did we choose a user experience metric?
+
+We could have chosen a customer growth metric for our North Star, but as an early-stage startup this felt premature. 
+
+PostHog is product-led, so the most important thing for us is knowing that engineers who use PostHog are discovering insights to improve their products. It validates our work and enhances our understanding of what our users need.
+
+When we conducted user cohort tests to decide on our North Star Metric, we found Discoveries retained 2.3x more users than the alternative metric.
+
+For us, retaining users is the first step on the path to profitability and a metric that many different teams (e.g. engineering, developer relations and content) can support.
+
+## How to track your North Star metric
+
+Product analytics platforms are the ideal way to track your North Star metric, as they enable you to pull a variety of data into a single platform and correlate it to other activities. 
+
+Naturally, we think PostHog is the best product analytics platform available because it enables you to deploy analytics onto your own infrastructure. That means you don't need to rely on third parties which may be blocked by ad blockers or privacy features, nor do you need to send any data to a third-party platform. This approach increases the accuracy of your data, minimizes the risk of security breaches and gives you greater control. 
+
+Of course, there are plenty of other product analytics tools - including [open-source analytics platforms](https://posthog.com/blog/best-open-source-analytics-tools), as well as closed-source alternatives such as MixPanel, Amplitude and Pendo. 
+
+> New to product analytics? Check out our [introduction to product analytics](https://posthog.com/blog/what-is-product-analytics), or [request a demo with one of our engineers](https://posthog.com/book-a-demo) to get started with PostHog. 
+
+## Why you shouldn't use Google Analytics to track your North Star metric
+
+Google Analytics is good for a lot of things, especially if you want to track basic web metrics such as pageviews or vanity metrics such as bounce rate. However, as a _web_ analytics platform it is limited to tracking only specific metrics for websites. It can't track other types of events, nor does it offer the same breadth of analysis tools that a product analytics platform like PostHog does. 
```

Comment:
```suggestion
Google Analytics is good for a lot of things, especially if you want to track basic web metrics such as pageviews or  bounce rate. However, as a _web_ analytics platform it is limited to tracking only specific metrics for websites. It can't track other types of events, nor does it offer the same breadth of analysis tools that a product analytics platform like PostHog does. 
```